README.md

```diff
@@ -17,26 +17,29 @@ $  sudo apt-get install mysql-server libmysqlclient-dev imagemagic
 If you want to run the unit tests you need to install PhantomJS directly from this link:
 http://phantomjs.org/download.html
 
-Setup the app as with most rails apps:
+
+Setup
+-----
+
+###With Docker:
 ```
-$  cp config/application.yml.example config/application.yml
+$ cp .env.example .env
+$ cp docker-compose.yml.example docker-compose.yml
+$ docker-compose up --build -d
+$ docker-compose exec app rake db:setup
+$ docker-compose exec app rake user:populate_users
```

Comment:
This command is failing to populate any users, I believe for reasons unrelated to this PR (email can't be blank, and nothing is in `profile.css('.email')`). Not sure if we want to support this command anymore - I don't think it needs to block merge but I think we should either open a ticket to fix it or take it out of the readme.

Reply:
Possibly both!

Reply:
Was (this PR)[https://github.com/EFForg/eff_fab/pull/88] related?